modules/core/js/src/main/scala/lucuma/core/math/geom/svg/package.scala

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
Should the file be moved as well?

Reply:
Ooops, I overlooked the js/jvm files. Moving them too.